src/transformers/models/idefics/configuration_idefics.py

```diff
@@ -114,9 +114,13 @@ def __init__(
         freeze_lm_head=False,
         freeze_vision_layers=True,
         freeze_vision_module_exceptions=[],
+        vision_model_params="{}",
+        vision_config=None,
         vision_model_name="google/vit-base-patch16-224",
         vision_embed_dim=768,
         vision_image_size=224,
+        vision_intermediate_size=37,
+        vision_patch_size=2,
```

Comment:
```suggestion
        vision_intermediate_size=5120,
        vision_patch_size=14,
```
this is what's used in 9b and 80b